tiledb/sm/serialization/query.cc

```diff
@@ -593,18 +629,41 @@ Status query_from_capnp(
     uint64_t* existing_offset_buffer_size = nullptr;
     void* existing_buffer = nullptr;
     uint64_t* existing_buffer_size = nullptr;
+    uint8_t* existing_validity_buffer = nullptr;
+    uint64_t* existing_validity_buffer_size = nullptr;
 
     auto var_size = schema->var_size(name);
+    auto nullable = schema->is_nullable(name);
```

Comment:
Same questions as above, is the nullable bool good enough or do we need to check if the capn'n proto object contains the nullable sizes?

Reply:
The `nullable` flag must supersede everything else. Even if we have validity buffer sizes in the message, we can't use them because the `set/get_buffer[*_nullable]` APIs will fail if the attribute is non-nullable.